packages/builder-core/src/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "outDir": "../build/",
+    "rootDirs": ["."],
+    "composite": true
+  },
+  "include": ["./**/*.ts"],
+  "exclude": []
+}
```

Comment:
let's move this file one level up

Reply:
I found that linting setup uses these both configs separately. It's possible to run the linter for src and for tests separately. I'm not sure if it's useful right away.

Reply:
builder-core/tsconfig.json compilation file was changed to compile everything instead of tests